doc/source/whatsnew/v0.21.0.txt

```diff
@@ -179,6 +179,7 @@ Sparse
 ^^^^^^
 
 
+- Bug in instantiating :class:`SparseSeries` from ``dict`` with or without ``index=`` kwarg (:issue:`16905`)
```

Comment:
If it doesn't matter whether `index` is passed in, why mention it in the description?